src/ProgressOnderwijsUtils/SchemaReflection/DatabaseDescription.cs

```diff
@@ -75,12 +76,35 @@ public sealed record DataByTableId(
         Dictionary<DbObjectId, CheckConstraintSqlDefinition[]> CheckContraints,
         Dictionary<DbObjectId, DmlTableTriggerSqlDefinition[]> Triggers);
 
+    sealed record SqlExpressionDependencies(DbObjectId referencing_id, DbObjectId referenced_id) : IWrittenImplicitly;
+
     public static DatabaseDescription LoadFromSchemaTables(SqlConnection conn)
     {
         var tables = DbNamedObjectId.LoadAllObjectsOfType(conn, "U");
         var views = DbNamedObjectId.LoadAllObjectsOfType(conn, "V");
-        var columnsByTableId = DbColumnMetaData.LoadAll(conn);
-        return new(tables, views, columnsByTableId, ForeignKeyColumnEntry.LoadAll(conn), CheckConstraintSqlDefinition.LoadAll(conn), DmlTableTriggerSqlDefinition.LoadAll(conn), DefaultValueConstraintSqlDefinition.LoadAll(conn), ComputedColumnSqlDefinition.LoadAll(conn), SequenceSqlDefinition.LoadAll(conn));
+        var dependencies = SQL(
+            $@"
+                select
+                    sed.referencing_id
+                    , sed.referenced_id
+                from sys.sql_expression_dependencies sed
+                where 1=1
+                    and sed.referencing_id in {views.ArraySelect(view => view.ObjectId)}
```

Comment:
Ik denk niet dat deze inperking zinnig is - maakt het vast niet sneller, en wie weet zijn andere objects ook nog ooit interessant.